src/Guide/Api/Server.hs

```diff
@@ -71,13 +71,20 @@ type FullApi =
 
 fullServer :: DB -> Server FullApi
 fullServer db =
-  toServant (apiServer db) :<|>
+  api db :<|>
   swaggerSchemaUIServer doc
   where
     doc = toSwagger (Proxy @Api)
             & info.title   .~ "Aelve Guide API"
             & info.version .~ "alpha"
 
+-- | Convertor from api with Guider to api with Handler via hoistServer.
+api :: DB -> Server Api
+api db = hoistServer proxy guiderToHandler (toServant (guiderServer db))
+
+proxy :: Proxy Api
```

Comment:
You don't need this definition, just write `hoistServer (Proxy @Api) ...`. You'll need to enable `TypeApplications` for that.